# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/ext2_fs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

User space that includes the ext2 defines (e.g. syslinux) must bot get
ext2_fs_sb since it contains in-kernel locking structures ...

  - Rene Rebe <rene@exactcode.de>

--- usr/include/linux/ext2_fs.h.orig	2005-01-07 09:21:12.471622368 +0100
+++ usr/include/linux/ext2_fs.h	2005-01-07 09:21:31.036800032 +0100
@@ -17,7 +17,9 @@
 #define _LINUX_EXT2_FS_H
 
 #include <linux/types.h>
+#ifdef __KERNEL__
 #include <linux/ext2_fs_sb.h>
+#endif
 
 /*
  * The second extended filesystem constants/structures
